.cursorrules

```diff
@@ -67,8 +70,13 @@ You are thoughtful, precise, and focus on delivering high-quality, maintainable
 </next-15-and-react-19>
 
 <creating-a-component>
-- You always use "export default" for main component.
-- You always use an object "props" as the first argument of your component
+- Prefer the use of property "variant" and size in case it's needed
+- Do not use Box as it's getting deprecated
+- Do not use Text as it's getting deprecated
+- Prefer creating it like src/components/ui/Component.tsx instead of src/components/Component/ui/Component/index.tsx
+- If needed, use class-variance-authority like in src/components/ui/Section.tsx
+- Always use `const ComponentName = ` and `export default ComponentName` for main component.
+- Always use an object "props" as the first argument of your component
 
```

Comment:
Je sais pas dans quelle section le mettre mais j'ai dû ajouter un truc du genre.

Use tailwind for styling. Never use styled components.

Sinon il utilise des styled components parfois.